src/Console/Commands/ModelBackpackCommand.php

```diff
@@ -57,7 +57,7 @@ protected function getStub()
      */
     protected function getDefaultNamespace($rootNamespace)
     {
-        return $rootNamespace;
+        return $rootNamespace.'\Test';
```

Comment:
Did you mean to type `Models` here? 

```suggestion
        return $rootNamespace.'\Models';
```

Or... what's happening? I don't see how this would make models be generated into `app\Models`, sorry if I'm being dumb 😅 Also, does this also account for the model namespace - when it's used in the generated CrudController?

Reply:
So when using the `Backpack:Crud` it generates fine. It does correct the namespace and sorry for the 'Test' it was me testing things and i copy/pasted. That's my bad. :) That was my other thought was adding an option to use older versions like a `1` for use older locations, but i'm not sure how many people are still using the older versions.